force-app/main/default/classes/ProgramSelector.cls

```diff
@@ -16,11 +22,33 @@ public with sharing class ProgramSelector {
         List<ProgramCohort__c> programCohorts = [
             SELECT Id, Name
             FROM ProgramCohort__c
-            WHERE Program__c = :programId
+            WHERE Program__c = :programId AND Status__c IN :allowedProgramCohortStatuses
             ORDER BY Name ASC
         ];
 
         return Security.stripInaccessible(AccessType.READABLE, programCohorts)
             .getRecords();
     }
+
+    public Set<String> getAllowedProgramCohortStatusValues() {
```

Comment:
Should this be private? And do we need to add a test case to validate getProgramCohortsFromProgramId now only returns active? Do we want to consider modifying the method name to getActiveProgramCohorts?